docs/update_castxml_version.rst

```diff
@@ -30,6 +64,9 @@ Available CastXML archives can be found `here <https://data.kitware.com/#folder/
     Updating tests/test_distribution.py
     Updating tests/test_distribution.py - done
 
+   *Note*: Ensure that the ``Win32_binary`` variables still are set to the
+   content of the ``Win64_binary`` variable.  It may be set to NULL.
```

Comment:
Instead, let's fix the script:

https://github.com/CastXML/CastXML-python-distributions/blob/37c6977f009c34b9b891f2910cf5207c11d90f30/scripts/update_castxml_version.py#L93-L94

In the commit message updating the script, reference issue #5  

Suggested change:

```diff
-      set(win32_binary_url    "NA")  # Windows 32-bit binaries not available
+      set(win32_binary_url    "${win64_binary_url}")  # See https://github.com/CastXML/CastXML-python-distributions/issues/5
-      set(win32_binary_sha512 "NA")
+      set(win32_binary_sha512 "${win64_binary_sha512}")
```


For some more context, see https://github.com/CastXML/CastXML-python-distributions/commit/e03b460b03032d00e00fcca27ae10548d22999a5#diff-aa166f56653254a5ea3d6db905d61992